program/src/state.rs

```diff
@@ -179,6 +208,7 @@ pub struct ProposalVote {
     ///
     /// * `true`: In favor.
     /// * `false`: Against.
+    // Jon: it might be worth creating an enum for the vote type, where 0 is "not voted"
```

Comment:
Does that mean someone can have three kinds of votes? How does one cast a "not voted"?

Reply:
You wouldn't cast a "not voted", but some governance systems make it permissionless to create the proposal vote account for others, and then voters can fill in their vote whenever they want without paying for rent. I'm mainly worried about `0` meaning `no`.

Reply:
As another use case, someone could also switch their vote to an abstention as a way to "cancel" their vote

Reply:
Added in #11.